Source/Core/DolphinQt/Settings/AudioPane.cpp

```diff
@@ -181,12 +181,18 @@ void AudioPane::LoadSettings()
 
   // Backend
   const auto current = SConfig::GetInstance().sBackend;
+  bool selection_set = false;
```

Comment:
Can we not just set the index to -1 here and be done with it?

Reply:
No, because the act of adding an item (or the act of adding the first item?) sets it back to 0.

Reply:
Oh I missed the `addItem` in the loop.